### DISCUSSION TITLE
V3 and Alpine CSP version

### DISCUSSION THREAD

**Discussion by m-lotze** [livewire/livewire#6468] [1. Help]
Is it possible to use livewire 3 with the alpine csp version - https://github.com/alpinejs/alpine/tree/main/packages/csp?

this does not work. Livewire still uses the bundled alpine:

```
import Alpine from '../../../alpine/packages/csp';
import { Livewire } from '../../vendor/livewire/livewire/dist/livewire.esm';

Livewire.start();
```

**Comment by joshhanley** [comment]
@m-lotze not at this stage, no. Also Alpine CSP isn't stable either.

**Comment by m-lotze** [comment]
Any update on this? Alpine CSP seems stable now. 

I think it would be enough to release a version with the alpine/csp bundled. Livewire version 2 works great with it.
Or maybe add some functionality to swap the bundled alpine version.

**Reply by kigathi-chege** [reply]
Hey, any updates on this?

**Reply by samods2** [reply]
Any updates?

**Comment by NullPointerReference** [comment]
As a Security Engineer I would strongly suggest addressing this, as 'unsafe-eval' is considered problematic in most security-sensitive contexts and a working solution seems feasible now.

**Comment by kigathi-chege** [comment]
This is really frustrating.
